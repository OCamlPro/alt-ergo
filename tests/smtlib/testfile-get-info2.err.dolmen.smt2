(set-logic ALL)

(declare-const x Int)

(assert (= (* x x) 4))

(get-info :reason-unknown)
(check-sat)