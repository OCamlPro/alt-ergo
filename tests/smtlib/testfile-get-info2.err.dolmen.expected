(error "Invalid (get-info :reason-unknown)")
